mos_tests/functions/common.py

```diff
@@ -543,3 +548,27 @@ def delete_volume_snapshot(cinder_client, snapshot):
         cinder_client.volume_snapshots.delete(snapshot)
         while snapshot in cinder_client.volume_snapshots.list():
             sleep(1)
+
+
+# Keys
+def is_key_exists(nova_client, key_name):
+    """ Check the presence of keys in the system
+        :param nova_client: Nova API client connection point
+        :param key_name: name of the keypair
+        :return True or False
+    """
+    return key_name in [k.name for k in nova_client.keypairs.list()]
```

Comment:
This function is similar to other such functions in this file (is_volume_exists, is_instance_exists, etc)